bin/crab.py

```diff
@@ -142,7 +142,9 @@ def log_exception(exc_type, exc_value, tback):
             sys.exit(-1)
         self.cmd = sub_cmd(self.logger, args[1:])
 
-        self.cmd()
+        returnDict = self.cmd()
+        if returnDict['commandStatus'] != 'SUCCESS':
+            sys.exit(1)
```

Comment:
We should return/raise back to `__main__` section instead of exit here.

Reply:
I agree. I put `sys.exit(1)` becasue raising ClientException did not seem adequate. In a way what I want is to make them end with exit code 1 some commands that until now reports a "failure" in the messages but return with exit code 0.
I will define a new exception for this